corehq/apps/registration/templates/registration/email/mobile_signup_reminder.html

```diff
@@ -0,0 +1,104 @@
+{% load i18n %}
+{% load hq_shared_tags %}
+<!doctype html>
+<html bgcolor="#f2f2f2" style="background-color: #f2f2f1; margin: 0; padding: 0; font-family: 'Helvetica Neue', 'Helvetica', 'Lucida Sans', 'Trebuchet MS', 'Arial', sans-serif;">
```

Comment:
FWIW, most of our HTML emails are pretty plain, I think the confirmation email is the only fancy one. I'm not opposed to styling this email in that way, but you should make a parent template and have both this and the confirmation email inherit from it.

Reply:
Mm, the spec said this one should mimic the activation email, so I just copied it here.

Inheritance sounds fine though